gato/policy/gato_policy.py

```diff
@@ -340,7 +347,9 @@ def tokenize_input_dicts(self, inputs: list):
             batch_tokens = torch.cat(
                 [
                     tokens for tokens in
-                    [text_tokens, image_tokens, continuous_tokens, discrete_tokens, separator_tokens, continuous_action_tokens, discrete_action_tokens]
+                    # Swap the order of image_tokens and text_tokens (GATO paper places test tokens before image tokens) for the image-caption task,
+                    # caption (which a text_tokens) should follow image instead of preceding image, so that image data can auto-regressively predict caption tokens
```

Comment:
I'm unsure if this is actually how it works. 
@daniellawson9999 care to comment?